appender/elasticsearch-rest/src/main/cfg/org.apache.karaf.decanter.appender.elasticsearch.rest-default.cfg

```diff
@@ -35,6 +35,9 @@ addresses=http://localhost:9200
 # index.event.timestamped=true
 # The index type.
 # index.type=decanter
+# The index filter: attribute1:value,attribute2:value...
+# if empty, allow to catch all collected events
+# index.filter=
```

Comment:
IMHO `index.filter` is not accurate. We don't filter the index, but actually the property names coming from the dispatcher. I would maybe rename as `properties.exclude` or `properties.filter`.

Reply:
Ok, I propose to use `properties.filter`

Reply:
As discussed, I would improve a bit to cover any use cases. I would propose:

```
properties.names.includes=*foo*,*bar*
properties.names.excludes=*xx*,*yy*
properties.values.includes=*foo*,*bar*
properties.values.excludes=*xx*,*yy*
```